src/Fixer/AttributeNotation/AttributeWithoutParenthesesFixer.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\AttributeNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author HypeMC
+ */
+final class AttributeWithoutParenthesesFixer extends AbstractFixer implements ConfigurableFixerInterface
```

Comment:
Before going further with the review, I would like to ask for changing fixer's name. I believe we need to add it to contributing guide, but:
- rules that make one-way change should be named after the state of the code after fix is applied, for example `no_useless_nullsafe_operator`.
- rules that are configurable and offer more than one way of standardising the code should be named after the area of responsibility, for example `nullable_type_declaration`

Name proposed here does not fit this, as it suggests that after applying fix attributes won't have parentheses. We need something more generic like `attribute_empty_parentheses` (the scope of the change), `attribute_syntax` (like `array_syntax`) or `attribute_style` (like `yoda_style`).

@keradus what do you think?

Anyway, I like the general idea of the fixer 👍.

Reply:
Hi, I agree with the naming, the reason I choose this name was so that it would (sort of) match with `NewWithBracesFixer`, but I'm guessing those older fixers might not have the best naming themselves, the word braces being used instead of parenthesis for one example. I've renamed the fixer to `AttributeEmptyParenthesesFixer` for the time being, if you decide on another name, please let me know and I'll update it.

Reply:
👍🏻 for @Wirone 

👍🏻 for new suggested name by @HypeMC